src/schema/objects/metadata.yaml

```
@@ -2481,6 +2481,32 @@ Resolution:
   - type: object
     additionalProperties:
       type: string
+RotationOrder:
+  name: RotationOrder
+  display_name: RotationOrder
+  description: |
+    Specify the sequence in which the elemental rotations are applied around the extrinsic axes.
+    A 3D rotation is represented about 3 different axes (Tait-Bryan convention). Must be one of:
+    "XYZ", "XZY", "YXZ", "YZX", "ZXY", "ZYX".
```

```suggestion
    A 3D rotation is represented about 3 different axes (Tait-Bryan convention).
```
We don't need to list enums in the descriptions.